templates/matching.jsx

```diff
@@ -43,6 +43,7 @@ export default function Matching(props) {
         }, index) => {
           const activeOption = _options.find(option => (option._itemIndex === _index) && option._isActive);
           const displayItemAsCorrect = (!_isEnabled && _shouldShowMarking && (_isCorrectAnswerShown || activeOption?._shouldBeSelected));
+          const _questionTitleId = `${_id}-matching-item-${_index}__title`;
```

Comment:
```suggestion
          const questionTitleId = `${_id}-matching-item-${_index}__title`;
```

Could you remove the `_` throughout please? It's only really loosely used to identify non-translatable properties in the JSON.

Reply:
Done